src/modules/dashboard/channelSelectModal.tsx

```diff
@@ -28,6 +30,18 @@ const ChannelSelectModal: FC<Props> = ({ phrasesByKey, showModal, setShowModal }
     })
     setInitialOrderStatusByMapping(selectedChannels)
     setShowModal(false)
+    const configuration = {
+      ...allState,
+      initialState: true,
+    }
+    try {
+      putEtrustedConfiguration(user?.access_token as string, {
```

Comment:
I still have a problem with it, sorry. If the user ist not there then you typecast undefined to string and tries to use undefined token for the call what will be 100% a fail. Why not to check before and if the user is not there then stop trying to send something to our backend? Same for interaction too.